### ISSUE TITLE
Investigate: Where get data for Booking screen and design GraphQL schema for that.

### ISSUE THREAD

**Issue by FilipMessa** [kiwicom/margarita#114]
We will need implementing the GraphQL schema for Booking screen.

Related Issues  #72, #73 

Check the Tequila API documentation and suggest the GraphQL schema.

*Pls don't start implemented it yet, just prepare the schema for CR first*

**Comment by FilipMessa** [comment]
If you will need to call `/booking` API, you must use a different `apikey` than current. There are different products with different policies, so check https://tequila.kiwi.com/portal/my-applications  and look for the `apikey` inside the *searchandbook* application.

**Comment by JoseDuSV** [comment]
Proposal for `/booking/check_flights` implementation from Booking API https://tequila.kiwi.com/portal/resources/booking_api.

In short, we need to check if itinerary is valid/bookable and receive actual itinerary data to show them inside booking screen header #72 .

Query name:
```
checkItinerary
```

Input:
```
input CheckItineraryInput {
  """Token received in search results"""
  bookingToken: String!

  """Number of bags for the booking"""
  bags: Int!

  """Number of children, adults and infants travelling"""
  passengers: PassengersInput!
}
```

Return:
```
type ItineraryValidity {
  """All segments successfully checked"""
  itineraryChecked: Boolean

  """Itinerary is valid (not sold-out or cancelled)"""
  itineraryValid: Boolean

  """Actual itinerary data"""
  itinerary: Itinerary
}
```

What do you think @FilipMessa @tbergq @jvaclavik @KarinaDavtyan @RobinCsl ?

**Comment by FilipMessa** [comment]
LGTM
I would like to ask, just for my information, bags can be zero, right? and passengers must be at least 1, right?

**Comment by JoseDuSV** [comment]
@FilipMessa Right, passengers number can be 1-9. And number of bags can be 0.

**Comment by jvaclavik** [comment]
And what if there are more types of the bag? For example with Ryanair you can have personal item (free), cabin bag (paid) and large bag (paid)

**Comment by JoseDuSV** [comment]
@jvaclavik According to description it seems to be related only to checked baggages `The number of bags for the booking, even if bags_price states that the first (or even second) checked baggage is free, it is necessary to request it`.

**Comment by jvaclavik** [comment]
> @jvaclavik According to description it seems to be related only to checked baggages `The number of bags for the booking, even if bags_price states that the first (or even second) checked baggage is free, it is necessary to request it`.

OK, in that case it looks fine

**Comment by JoseDuSV** [comment]
Closed by #633
